torch/onnx/symbolic.py

```diff
@@ -1396,6 +1399,11 @@ def to(g, self, *args):
         dtype = _get_const(args[0], 'i', 'dtype')
         # Layout and device are ignored
         return g.op("Cast", self, to_i=scalar_type_to_onnx[dtype])
+    elif len(args) == 6:
+        # aten::to(Tensor, ScalarType, Layout, Device, bool, bool, bool) -> Tensor
```

Comment:
That's some pretty bad boolean blindness

Reply:
Unfortunately all args except ScalarType are ignored in this function